easybuild/easyconfigs/u/unblur/unblur-1.0.2-foss-2018b.eb

```diff
@@ -0,0 +1,59 @@
+# Thomas Hoffmann, EMBL, Heidelberg, 2017/08  
+# THEMBL 201811: upgrade from foss/2017b to 2018b 
+easyblock = 'ConfigureMake'
+parallel=1
+name = 'unblur'
+version = '1.0.2'
+
+homepage = 'https://www.cl.cam.ac.uk/~mgk25/jbigkit/'
+description = """
+Unblur is used to align the frames of movies recorded
+ on an electron microscope to reduce image blurring due to beam-induced motion.
+ It reads stacks of movies that are stored in MRC/CCP4 format.
+ Unblur generates frame sums that can be used in subsequent image processing
+ steps and optionally applies an exposure-dependent filter to maximize the signal
+ at all resolutions in the frame averages. Movie frame sums can also be 
+calculated using Summovie, which uses the alignment results from a prior run of Unblur.
+ The idea for this procedure was originally developed by Brilot et al.
+ and Campbell et al [1] [2]. Unblur was written by Timothy Grant and 
+Alexis Rohou and is described in [3].
+1. Brilot, AF, Chen JZ, Cheng A, Pan J, Harrison SC, Potter CS, Carragher B,
+ Henderson R, Grigorieff N.  2012.  Beam-induced motion of vitrified 
+specimen on holey carbon film.. J Struct Biol. 177:630-637.
+2. Campbell, MG, Cheng A, Brilot AF, Moeller A, Lyumkis D, Veesler D,
+ Pan J, Harrison SC, Potter CS, Carragher B et al..  2012.  Movies of 
+ice-embedded particles enhance resolution in electron cryo-microscopy. 
+Structure. 20:1823–1828.
+3. Grant, T, Grigorieff N.  2015.  Measuring the optimal exposure for 
+single particle cryo-EM using a 2.6 Å reconstruction of rotavirus 
+VP6. eLife. 4(e06980):1-19."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
@garadar Toolchains older than `foss/2019a` have been deprecated for a while, see https://docs.easybuild.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains 

Would it be feasible to use a more recent toolchain for this?